docs/examples/sock-shop.md

```diff
@@ -4,52 +4,83 @@ sidebar_position: 8
+CustomResourceDefinition/alerts.notification.toolkit.fluxcd.io created
+CustomResourceDefinition/buckets.source.toolkit.fluxcd.io created
+CustomResourceDefinition/gitrepositories.source.toolkit.fluxcd.io created
+CustomResourceDefinition/helmcharts.source.toolkit.fluxcd.io created
+CustomResourceDefinition/helmreleases.helm.toolkit.fluxcd.io created
+CustomResourceDefinition/helmrepositories.source.toolkit.fluxcd.io created
+CustomResourceDefinition/imagepolicies.image.toolkit.fluxcd.io created
+CustomResourceDefinition/imagerepositories.image.toolkit.fluxcd.io created
+CustomResourceDefinition/imageupdateautomations.image.toolkit.fluxcd.io created
+CustomResourceDefinition/kustomizations.kustomize.toolkit.fluxcd.io created
+CustomResourceDefinition/providers.notification.toolkit.fluxcd.io created
+CustomResourceDefinition/receivers.notification.toolkit.fluxcd.io created
+Namespace/wego-system created
+ServiceAccount/wego-system/helm-controller created
+ServiceAccount/wego-system/image-automation-controller created
+ServiceAccount/wego-system/image-reflector-controller created
+ServiceAccount/wego-system/kustomize-controller created
+ServiceAccount/wego-system/notification-controller created
+ServiceAccount/wego-system/source-controller created
+ClusterRole/crd-controller-wego-system created
+ClusterRoleBinding/cluster-reconciler-wego-system created
+ClusterRoleBinding/crd-controller-wego-system created
+Service/wego-system/notification-controller created
+Service/wego-system/source-controller created
+Service/wego-system/webhook-receiver created
+Deployment/wego-system/helm-controller created
+Deployment/wego-system/image-automation-controller created
+Deployment/wego-system/image-reflector-controller created
+Deployment/wego-system/kustomize-controller created
+Deployment/wego-system/notification-controller created
+Deployment/wego-system/source-controller created
+NetworkPolicy/wego-system/allow-egress created
+NetworkPolicy/wego-system/allow-scraping created
+NetworkPolicy/wego-system/allow-webhooks created
 ◎ verifying installation
-✔ source-controller: deployment ready
-✔ kustomize-controller: deployment ready
 ✔ helm-controller: deployment ready
-✔ notification-controller: deployment ready
-✔ image-reflector-controller: deployment ready
 ✔ image-automation-controller: deployment ready
+✔ image-reflector-controller: deployment ready
+✔ kustomize-controller: deployment ready
+✔ notification-controller: deployment ready
+✔ source-controller: deployment ready
 ✔ install finished
+Deploy key generated and uploaded to git provider
+► Writing manifests to disk
+► Committing and pushing gitops updates for application
+► Pushing app changes to repository
+► Applying manifests to the cluster
 arete: /tmp/sock-shop>
 ```
 
-Once you see `install finished`, your cluster is ready to go with Weave GitOps.
+Once you see `► Applying manifests to the cluster`, your cluster is ready to go with Weave GitOps.
 
 ## Simple Deployment
```

Comment:
I believe simple deployment referred to when we weren't storing the automation (not that I am saying the deployment isnt simple). But perhaps given the subsequent section on '#deployment-with-gitops-resources-in-application-repository'. A more generic, "Deploying with Weave GitOps"? Also consider how the choice of header size impacts the right hand navigation structure. We could probably help break this up a bit more. Again not a pre-req for merge as this is a general improvement
<img width="295" alt="Screenshot 2021-11-23 at 13 37 22" src="https://user-images.githubusercontent.com/19860021/143034079-26714812-328c-4c6b-ab63-a3d9e2da1867.png">

Reply:
Change to this:
![Screenshot from 2021-12-02 10-14-05](https://user-images.githubusercontent.com/2802257/144479609-6b8d7a0e-4f3e-4673-9b81-121282a1cab1.png)